repose-aggregator/components/services/akka-http-client-service/akka-http-client-service-api/src/main/java/org/openrepose/core/services/serviceclient/akka/AkkaServiceClient.java

```diff
@@ -29,9 +29,11 @@
  */
 public interface AkkaServiceClient {
 
-    ServiceClientResponse get(String token, String uri, Map<String, String> headers) throws AkkaServiceClientException;
+    // TODO: Should backwards compatible get and post methods be added? If so, the commit to use the new methods can be reverted.
 
-    ServiceClientResponse post(String requestKey, String uri, Map<String, String> headers, String payload, MediaType contentMediaType) throws AkkaServiceClientException;
+    ServiceClientResponse get(String token, String uri, Map<String, String> headers, boolean checkCache) throws AkkaServiceClientException;
```

Comment:
Yeah, here's where you started making a bunch of unnecessary changes, overload the methods. Then most of our calls can just stay as they are, and when we need it specifically we can call for a non-cache answer. Also obviously implementation wise the one just calls into the other with true.